### DISCUSSION TITLE
Randomselect Pool Disabler for Characters

### DISCUSSION THREAD

**Discussion by NullAndVoid69** [ikemen-engine/Ikemen-GO#1433] [Feature Request]
### Is your feature request related to a problem? Please describe.

I have a character that I do not want to be randomly selected, as they are the final boss of arcade mode! (kinda like how the exclude parameter works) However, I still want that character to be SELECTABLE, as a secret character.

### Describe the solution you'd like

A parameter for character that allows them to be excluded from randomselect and modes like Survival, while also allowing them to be selectable. This could alternatively just be an option for the existing exclude or hidden parameters.

### Additional context

This feature is already in the engine, but it's only for stages. Implementing this feature for characters would, in this case, not be that hard, hypothetically. (also i had to shut down and restart my computer while writing this the first time so if this seems a bit rushed then you know why lol)

### Search terms

randomselect, random, secret

**Comment by ZyonZZ** [comment]
Your proposal is interesting...

In my case it wouldn't bother me, since for me RANDOM... means completely RANDOM...

But..
being an engine where you can have all kinds of characters even from different sagas and origins, series and characters of all kinds.
I would like the Random you could choose for example
a Random for KOF chars, (label only those that are in the KOF category... they will be selected... within this random)
and thus be able to create tags so that only if you are selecting that tag, the random respect among those characters that have it.

visually it would be already...
the normal slot for the defaults without configuring ""randomselect"" the normal slot that completely randomizes all the chars you have (the usual random select)

and the new function would be to put in the select.def the
  "randomselectADV" that this is not completely random but with rules....
When you position yourself in it you can choose between the tags you have created (or unlock the new tags why not)
With the new built-in buttons you would move between the labels, seeing the initials on the portal, "Kof, SF, DBZ, GuiltyGears, even ALL" etc.
After scrolling with the new "D" and "W" buttons, you choose SF..
and start the random roulette and only choose the characters with the SF tag

It's something similar to what you say about omitting characters in the random
With this system, if the character does not have a tag, it is simply omitted.
